grid/social-media/posts-lookup/maps/twitter.suma

```diff
@@ -196,3 +260,11 @@ operation MapUser {
 
   return author
 }
+
+operation GetSearchRecentTweetsBucket {
+  return "Search Recent Tweets"
```

Comment:
Watch out for bucket's string capitalization, for consistency. Assuming these buckets are used programmatically, they must be consistent across maps from single provider.
```suggestion
  return "Search recent tweets"
```

Reply:
OK I will go with lover case. First I wanted to use exactly the same value as is the name of resource documented in Twitter rate limit table. But in case of search there are two levels:
![image](https://user-images.githubusercontent.com/5206165/152311656-c0a51f3a-9d7f-4b28-9f77-82fc1d3e01da.png)

Reply:
fixed